2020-01-Winter/1_linked_lists/list-palindrome/question.txt

```diff
@@ -0,0 +1,21 @@
+
+Given a singly unsorted linked list of size N of integers. 
+Your task is to check if the given linked list is palindrome or not.
+
+Input:
+A singly usorted linked list of N integers 
+1 <= N <= 10
+
+Output:
+1 if the linked list is a palindrome, 0 otherwise
+
+
+
+public class question
+{
+    // Function to check if linked list is palindrome
+    boolean isPalindrome(Node head) 
+    {
+        //Your code here
+    }    
+}
```

Comment:
We actually have a file for the problem statement; it is the `README.md` file [in the directory above](https://github.com/UWB-ACM/CTCI/tree/master/2020-01-Winter/1_linked_lists). Can you add this content to the readme under Problem 1?

Also, in the README, there is a section for the problem's solution, where we typically add the source code for the solution method and a brief explanation of the reasoning behind it. For your brief explanation, the block comment in [solution.java](https://github.com/UWB-ACM/CTCI/blob/1459f7f3c8a775cf46aab139425ed8f5ae52e50a/2020-01-Winter/1_linked_lists/list-palindrome/solution.java) should be perfectly sufficient.